codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2RemoteFlowController.java

```diff
@@ -214,9 +218,41 @@ public ChannelFuture sendFlowControlledFrame(ChannelHandlerContext ctx, Http2Str
     }
 
     @Override
-    public ChannelFuture lastFlowControlledFrameSent(Http2Stream stream) {
-        FlowState state = state(stream);
-        return state != null ? state.lastNewFrame() : null;
+    public ChannelFuture sendHeaders(ChannelHandlerContext ctx, Http2Stream stream, Http2Headers headers,
+            int streamDependency, short weight, boolean exclusive, int padding,
+            boolean endOfStream, ChannelPromise promise) {
+      boolean isFirst = !sentFrame;
```

Comment:
I think I can use stream.state() == IDLE to achieve the same goal. i.e move the stream state out of IDLE after writing the first headers rather than before